### DISCUSSION TITLE
Question on upgrade from 3.2.1 to 3.5.1 helmchart

### DISCUSSION THREAD

**Discussion by Krrish438** [ceph/ceph-csi#3539] [Q&A]
Hi,

We are running few k8s cluster with helm version 3.2.1. we have pvc created with 3.2.1 helm chart and they are in use from long time.

we wanted to upgrade helm chart to 3.5.1.  Is there any risk with mounts go stale or go into error state?? We have tested this helm upgrade and no issues observed. But we just wanted to understand if there are any risks in upgrading. Please let me know.

BRs
Kishore

**Comment by Madhu-1** [comment]
v3.5 is an unsupported version. The best is to update some test clusters and see if everything works.
